deployment/HDI-Spark/Notebooks/install-interactive-notebook.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+#
+# Install Livy and SparkMagic on Head node
+#
+
+set +e
```

Comment:
Btw, just want to make sure that `set +e` is what you want.  It will ignore errors that come up in the script.

Reply:
I think this is fine. I have capture the parameter missing error below. It will show below error message in stdout if sparkdotnet version is null. Let me know if you see any other issues. Thanks!

> Error: Sparkdotnet version parameter is missing...
('Start downloading script locally: ', u'https://sparkdotnetrepl.blob.core.windows.net/notebooks/install-interactive-notebook.sh')
Fromdos line ending conversion successful
('Unexpected error:', "('Execution of custom script failed with exit code', 1)")
Removing temp location of the script

Command failed after 1 tries

Reply:
Here you manually call `exit 1` after checking for a parameter.  I'm more concerned about other unforeseen errors that may occur, like the storage account or nuget.org somehow not being available.  Would scenarios like this give a false positive to the user that the script was successfully run ?

Reply:
You are right. How about `set -e` instead?

Reply:
Sure.

Reply:
Changed.